src/coreclr/vm/eventtrace.cpp

```diff
@@ -6591,7 +6579,7 @@ VOID ETW::MethodLog::SendMethodEvent(MethodDesc *pMethodDesc, DWORD dwEventOptio
     ULONGLONG ullMethodStartAddress=0, ullColdMethodStartAddress=0, ullModuleID=0, ullMethodIdentifier=0;
     ULONG ulMethodSize=0, ulColdMethodSize=0, ulMethodToken=0, ulMethodFlags=0, ulColdMethodFlags=0;
```

Comment:
I think we can also remove the `ullColdMethodStartAddress` and `ulColdMethodSize` locals.